### PR TITLE
Add removal of keep awake events for cron history

### PR DESCRIPTION
**Pull request by eadenink**
There is one issue: when interval is more than a minute cron start rounding dates, so for default 120 seconds available 2 latest logs, but for 10 seconds interval only 1

### CODE REVIEW

File: example/packages/oplog-keep-awake/server.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import prepareProperSchedule from './utils/prepareProperSchedule';
 
 const defaultSettings = {
   keepAwakeCollectionName: 'keepAwake',
-  keepAwakeUpsertIntervalSeconds: 120,
+  keepAwakeUpsertIntervalSeconds: 10,
```
Comment (kolyasya) [inline example/packages/oplog-keep-awake/server.js]: @eadenink you changed a default interval here. We publish the package code from this directory

File: example/packages/oplog-keep-awake/server.js [hunk 2]
```diff
@@ -37,16 +37,23 @@ class OplogKeepAwake {
...
+        name: cronOperationName,
         schedule(parser) {
           return prepareProperSchedule({
             intervalSeconds: packageSettings.keepAwakeUpsertIntervalSeconds,
             parser,
           });
         },
         job() {
+          cronHistoryCollection.remove({ 
+            name: cronOperationName,
```
Comment (kolyasya) [inline example/packages/oplog-keep-awake/server.js]: @eadenink shouldn't the query be simpler? We can just use regex for `kolyasya:oplog-keep-awake` string

File: example/packages/oplog-keep-awake/server.js [hunk 3]
```diff
@@ -37,16 +37,23 @@ class OplogKeepAwake {
...
         schedule(parser) {
           return prepareProperSchedule({
             intervalSeconds: packageSettings.keepAwakeUpsertIntervalSeconds,
             parser,
           });
         },
         job() {
+          cronHistoryCollection.remove({ 
+            name: cronOperationName, 
+            finishedAt: { $lte: new Date(new Date() - packageSettings?.keepAwakeUpsertIntervalSeconds * 1000) }
```
Comment (kolyasya) [inline example/packages/oplog-keep-awake/server.js]: @eadenink need do add a human readable comment here with the date explanation

**Review by kolyasya — CHANGES_REQUESTED** [review]

File: example/packages/oplog-keep-awake/server.js (hunk 2 above)
Comment (eadenink) [reply example/packages/oplog-keep-awake/server.js]: Anyway we use this message in cron so I think it's reasonable to use this message also for removal

Comment (kolyasya) [reply example/packages/oplog-keep-awake/server.js]: @eadenink what if we adjust settings before the next clean-up CRON start? Old log will not be removed from DB

File: example/packages/oplog-keep-awake/server.js [hunk 4]
```diff
@@ -37,16 +37,23 @@ class OplogKeepAwake {
...
         schedule(parser) {
           return prepareProperSchedule({
             intervalSeconds: packageSettings.keepAwakeUpsertIntervalSeconds,
             parser,
           });
         },
         job() {
+          cronHistoryCollection.remove({ 
+            name: cronOperationName, 
+            finishedAt: { $lte: new Date(new Date() - packageSettings?.keepAwakeUpsertIntervalSeconds * 1000) } // Delete old package logs from cron history(from current time we subtract cron interval in ms)
```
Comment (kolyasya) [inline example/packages/oplog-keep-awake/server.js]: @eadenink add comment above the line please

**Pull request merged**